ZenPacks/ZenSystems/ApcUps/zenpack.yaml

```diff
@@ -0,0 +1,169 @@
+            type: SNMP
+            datapoints:
+              remainingCapacity:
+                rrdmax: 100
+                aliases: {remainingCapacity__pct: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.2.2.1.0
+          remainingTime:
+            type: SNMP
+            datapoints:
+              remainingTime:
+                aliases: {remainingTime__mins: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.2.2.3.0
+        graphs:
+          Actual Voltage:
+            units: Volts
+            graphpoints:
+              actualVoltage:
+                dpName: actualVoltage_actualVoltage
+          Remaining Capacity:
+            units: '%'
+            miny: 0
+            maxy: 100
+            custom: --alt-y-grid
+            graphpoints:
+              remainingCapacity:
+                dpName: remainingCapacity_remainingCapacity
+          Remaining Time:
+            units: mins
+            graphpoints:
+              remainingTime:
+                dpName: remainingTime_remainingTime
+                rpn: 6000,/
+      ApcUpsAdvanced:
+        description: ''
+        targetPythonClass: Products.ZenModel.Device
+        datasources:
+          upsAdvBatteryCapacity:
+            type: SNMP
+            datapoints:
+              upsAdvBatteryCapacity:
+                aliases: {ups_adv_battery_capacity__pct: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.2.2.1.0
+          upsAdvOutputFrequency:
+            type: SNMP
+            datapoints:
+              upsAdvOutputFrequency:
+                aliases: {ups_adv_output_frequency__hz: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.4.2.2.0
+          upsAdvOutputLoad:
+            type: SNMP
+            datapoints:
+              upsAdvOutputLoad:
+                aliases: {ups_adv_output_load__pct: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.4.2.3.0
+          upsAdvOutputVoltage:
+            type: SNMP
+            datapoints:
+              upsAdvOutputVoltage:
+                aliases: {ups_adv_output_voltage__vac: null}
+            oid: 1.3.6.1.4.1.318.1.1.1.4.2.1.0
+        graphs:
+          Battery Capacity:
+            units: '%'
+            graphpoints:
+              upsAdvBatteryCapacity:
+                dpName: upsAdvBatteryCapacity_upsAdvBatteryCapacity
+          Output Frequency:
+            units: Hz
+            graphpoints:
+              upsAdvOutputFrequency:
+                dpName: upsAdvOutputFrequency_upsAdvOutputFrequency
+          Output Load:
+            units: '%'
+            graphpoints:
+              upsAdvOutputLoad:
+                dpName: upsAdvOutputLoad_upsAdvOutputLoad
+          Output Voltage:
+            units: VAC
+            graphpoints:
+              upsAdvOutputVoltage:
+                dpName: upsAdvOutputVoltage_upsAdvOutputVoltage
+
+classes:
+  DEFAULTS:
+    base: [zenpacklib.Component]
+
+  ApcUpsDevice:
+    base: [zenpacklib.Device]
+    label: Device
+    impacted_by: [apcUpsBatterys]
```

Comment:
This should be:
    dynamicview_views: [service_view]
    dynamicview_relations:
      impacted_by: [apcUpsBattery]